lib/usd/ui/layerEditor/saveLayersDialog.cpp

```diff
@@ -486,18 +487,41 @@ void SaveLayersDialog::getLayersToSave(
     for (const auto& layerInfo : StageLayersToSave._anonLayers) {
         _stageLayerMap.emplace(std::make_pair(layerInfo.layer, stageName));
     }
+
     for (const auto& dirtyLayer : StageLayersToSave._dirtyFileBackedLayers) {
         _stageLayerMap.emplace(std::make_pair(dirtyLayer, stageName));
     }
 
+    // We do not allow saving layers in any of the following conditions:
```

Comment:
SaveLayerDialog will no longer list anonymous or file backed layers that are system locked, whether directly or by the extension of the parent layers.